ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
```

Comment:
The class name change from `.yaml_content_diagnosisContent` to `.yaml_content_diagnosisBody` may require updating references elsewhere in the codebase.